lib/system/containers.toit

```diff
@@ -64,12 +68,18 @@ class Container extends ServiceResourceProxy:
     if not code: throw "CLOSED"
     return code
 
+  on_stopped lambda/Lambda? -> none:
+    on_stopped_ = lambda
+    if not lambda or not result_.has_value: return
+    code := result_.get
+    if not code: throw "CLOSED"
+    lambda.call code
+
   on_notified_ code/int -> none:
```

Comment:
Does "on_notified_" mean that the container is stopped?

Reply:
Not necessarily, but here it does. In general `on_notified_` is called on the resource proxy when the `notify` method has been called on the resource.